tests/api-watch-diff/test.ml

```diff
@@ -134,14 +134,6 @@ let%expect_test "Modifying a simple type" =
   Format.printf "%a" pp_diff_list result;
   [%expect {|[Any]|}]
 
-let modify_type_in_value_signature =
```

Comment:
Why was this removed here? I'm assuming it wasn't used anymore and that's a left over from a previous PR but I'm still curious why the CI missed it if that's indeed the case!

Reply:
Yes, it is indeed a leftover from #43 (I noticed it a few days ago and thought of removing it in the next PR) and even I was surprised that an unused variable wasn't caught by the CI. It may be a bug in the CI!

Reply:
Did you get the unused value warnings when running the tests locally?

Reply:
No, I didn't.

Reply:
Might be something about how expect_test are run. It's possible this file is never actually built in a regular way and the warnings go unnoticed.